scpdt/plugin.py

```diff
@@ -19,12 +23,31 @@ def pytest_configure(config):
     doctest.DoctestModule = DTModule
     doctest.DoctestTextfile = DTTextfile
 
+def pytest_unconfigure(config):
+    if len(copied_files) > 0:
+        try:
+            for filepath in copied_files:
+                os.remove(filepath)
+        except FileNotFoundError:
+            pass
```

Comment:
ISTM this is never invoked. This patch should bring new failures,

```diff
diff --git a/scpdt/plugin.py b/scpdt/plugin.py
index 51a7653..09974bb 100644
--- a/scpdt/plugin.py
+++ b/scpdt/plugin.py
@@ -18,12 +18,13 @@ def pytest_configure(config):
     """
     Allow plugins and conftest files to perform initial configuration.
     """
-
+    raise ValueError
     doctest._get_checker = _get_checker
     doctest.DoctestModule = DTModule
     doctest.DoctestTextfile = DTTextfile
 
 def pytest_unconfigure(config):
+    raise ValueError
     if len(copied_files) > 0:
         try:
             for filepath in copied_files:
```

the number of failures is exactly the same as without it, at least when using
```
$ pytest --pyargs scpdt
```